### PR TITLE
Improve readme and organize package.json

### PR DESCRIPTION
**Pull request by grikomsn**
Hello there 👋🏻

Many thanks for the awesome plugin, thought I give back by updating the readme and a little bit of organizing the `package.json` structure for readability. Collapsible content below are image previews of the readme before and after updating.

<details>
<summary><b>Screenshot preview</b></summary>
<br />

| Before | After |
| ------ | ----- |
| image | image |

</details>

Let me know if there's anything I shoud revert or change.

### CODE REVIEW

**Comment by jbolda** [comment]
I haven't had a chance to fully review, but I wanted to say thanks @grikomsn 😊

File: LICENSE [hunk 1]
```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2018 Jacob Bolda
+Copyright (c) 2020 Jacob Bolda
```
Comment (jbolda) [inline LICENSE:3]:
```suggestion
Copyright (c) 2018, 2019, 2020 Jacob Bolda
```
We want to keep the original copyright year and update it for every year that we made a material change. (Which happens to be all of them.)

File: package.json [hunk 2]
```diff
@@ -1,32 +1,33 @@
...
-    "all-contributors-cli": "6.14.0"
-  }
+  "keywords": [
+    "gatsby",
+    "gatsby-plugin",
+    "gatsby-source",
+    "gatsby-source-airtable",
+    "airtable"
+  ],
+  "license": "MIT"
```
Comment (jbolda) [inline package.json:32]: Is there a guide that you are using to sort these? I would expect these make most sense under author.

File: README.md [hunk 3]
```diff
@@ -1,17 +1,44 @@
+<!-- markdownlint-disable MD033 MD041 -->
+<div align="center">
+
+![gatsby-source-airtable](./header.png)
```
Comment (jbolda) [inline README.md:4]: Lovely!

**Comment by grikomsn** [comment]
> Is the markdown wraps coming from Prettier? I am fine switching it to wrapped text, but I would like something "standard" if we are going to do it.

I manually wrapped it, but Prettier using `--prose-wrap` works better. Forgot the option exists. 😅

File: package.json (hunk 2 above)
Comment (grikomsn) [reply package.json:32]: There's no guide for sorting the keywords

**Comment by jbolda** [comment]
I just don't want to manually wrap it from now and into the future 😄 . I just updated prettier if you wanted to merge in from master and add a command to take care of running prettier for us.

**Comment by grikomsn** [comment]
Should I submit another PR with the suggested changed and with the latest upstream commits?

**Comment by grikomsn** [comment]
Since there are new changes from upstream, I'll close this PR and create another from the latest upstream commits (#180).

**Comment by jbolda** [comment]
I think we could have taken care of it in this PR, but no worries. Do you plan to open another PR with the readme updates? 🙏 That logo was 🔥 .

**Comment by grikomsn** [comment]
Definitely will create another PR after #180 is merged ✨

**Pull request closed without merging**